src/Translators/Rules.php

```diff
@@ -20,7 +21,43 @@ public static function fromColumn(Column $column)
 
         // hack for tests...
         if (in_array($column->dataType(), ['string', 'char', 'text', 'longText'])) {
-            $rules = array_merge($rules, ['string']);
+            $rules = array_merge($rules, [self::overrideStringRuleForSpecialNames($column->name())]);
+        }
+
+        if (in_array($column->dataType(), [
```

Comment:
There's probably an opportunity to get a bit more precise. For example, any of the _integer_ data type columns could use `integer` validation rule instead of `numeric`.